airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -101,8 +101,8 @@ def __init__(
         super().__init__(base_log_folder, filename_template)
         self.closed = False
 
-        self.client = elasticsearch.Elasticsearch(host.split(";"), **es_kwargs)  # type: ignore[attr-defined]
-
+        self.client = elasticsearch.Elasticsearch(host, **es_kwargs)  # type: ignore[attr-defined]
+        # in airflow.cfg, host of elasticsearch has to be http://dockerhostXxxx:9200
```

Comment:
May I know what error do we see if the protocol is not included in the set value?

Reply:
In ES 8, when constructing a new ElasticSearch client, a full URL including http / https is expected, and you will get a `ValueError` when calling the `__init__` like below: 

<img width="948" alt="image" src="https://github.com/apache/airflow/assets/43698890/342d4516-efd3-426c-99ce-3a396cdfa89f">